addons/website/views/assets.xml

```diff
@@ -148,5 +149,10 @@
 
     <script type="text/javascript" src="/website/static/src/js/widgets/ace.js"/>
 </template>
+<template id="website.user_custom_css" name="user custom css">
+    <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/user_custom_bootstrap_overridden.scss"/>
+    <!-- Custom empty file for user custom rules -->
+    <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/user_custom_rules.scss"/>
+</template>
```

Comment:
Those should not be in a dedicated assets, just forced at the end of their current respective assets. You can do that with an inherited view with a high sequence.

Also, user_custom_bootstrap_overridden.scss cannot be put there like that, it is meant to override bootstrap variables before bootstrap rendering, it cannot be done after, even less in a dedicated bundle which... does not contain any scss. It has to be before other bootstrap variables. Clearly, the two files should stay in the same bundle. Test your work with the HTML/CSS editor: the suggestion in the files should still work.

Reply:
by "the two files", you mean `bootstrap_overriden.scss` and `user_custom_bootstrap_overridden.scss`?